### PR TITLE
Bug 1200078 - [browser] Remove deprecated mozL10n.get

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1200078

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5532f14cddd29fb6f4d806dd2a617a4d1026bb87

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eb1b484eabe6b87a94d453e48e9766ca6a8c39a1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31ecddf89bab4a984c8053e44c533eca2238797d

File: tv_apps/browser/js/toolbar.js [hunk 1]
```diff
@@ -167,8 +166,9 @@ var Toolbar = {
     engineName = engineName.charAt(0).toUpperCase() + engineName.slice(1);
     this.searchInput.placeholder = engineName;
     // tooltip
-    this.searchInput.dataset.tips =
-      engineName + ' ' + _('WB_LT_TIPS_SEARCH_BAR');
+    navigator.mozL10n.formatValue('WB_LT_TIPS_SEARCH_BAR', val => {
+      this.searchInput.dataset.tips = val;
```
Comment (begeeben) [inline tv_apps/browser/js/toolbar.js]: It seems the ```engineName``` is missing here.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=62f6b41b8775fde10ef832fdf8c08154601ece98

**Pull request merged**